### PR TITLE
fix(ui5-table): fix row navigation and focus handling

### PR DESCRIPTION
**Pull request by ilhan007**
Fixes: https://github.com/SAP/ui5-webcomponents/issues/861

### CODE REVIEW

File: packages/main/src/TableRow.js [hunk 1]
```diff
@@ -80,6 +107,11 @@ class TableRow extends UI5Element {
 			return;
 		}
 
+		this.cells.forEach(cell => {
+			cell.removeEventListener("click", this.fnOnCellClick);
```
Comment (MapTo0) [inline packages/main/src/TableRow.js]: you better fire some event from the cell and rely on bubbling so you can catch it in the Row

File: packages/main/src/Table.js [hunk 2]
```diff
@@ -210,6 +210,10 @@ class Table extends UI5Element {
 		}
 	}
 
+	onColumnHeaderClick() {
+		this.getDomRef().querySelector(`#${this._id}-columnHeader`).focus();
```
Comment (MapTo0) [inline packages/main/src/Table.js]: i wish we make this part of the item navigation but its fine for now

**Comment by MapTo0** [comment]
also a test would be cool :D

File: packages/main/src/Table.js (hunk 2 above)
Comment (ilhan007) [reply packages/main/src/Table.js]: I can try to cancat it with this.rows, it might work

File: packages/main/src/TableRow.js (hunk 1 above)
Comment (ilhan007) [reply packages/main/src/TableRow.js]: Done

**Pull request merged**